R/severalChunks.R

```diff
@@ -32,4 +32,4 @@ calcClosedChamberFluxForChunks <- function(
 	ans
 }
 
-
+# TTR: Why not give it different areas by chunk, as well?
```

Comment:
@TTRademacher: The volume by chunk was added for portable chambers sitting on an installed ring. The rings change the volume of headspace but not the surface area. Differences aris because of insertion depth or uneven soil surface.